Modules/System/Time Zone Management/Permissions/TimeZoneMgtObjects.PermissionSet.al

```diff
@@ -0,0 +1,10 @@
+permissionset 8720 TimeZoneMgtObjects
+{
+    Caption = 'Time Zone Management - Objects';
+    Assignable = false;
+    Permissions = codeunit "DateTime Offset" = X,
```

Comment:
Let's add a blank line here between assignable and permissions - just to make the permissions portion stand out a little more.